Index: nvflare/app_opt/flower/applet.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/nvflare/app_opt/flower/applet.py b/nvflare/app_opt/flower/applet.py
--- a/nvflare/app_opt/flower/applet.py	(revision a5476ae9c62c0a02726c13d255baa733926ff180)
+++ b/nvflare/app_opt/flower/applet.py	(date 1737625607231)
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import os
 import time

 from nvflare.apis.fl_context import FLContext
@@ -57,7 +58,8 @@
         job_id = fl_ctx.get_job_id()
         custom_dir = ws.get_app_custom_dir(job_id)
         app_dir = ws.get_app_dir(job_id)
-        cmd = f"flower-supernode --insecure --grpc-adapter --superlink {addr} {custom_dir}"
+        rel_custom_dir = os.path.relpath(custom_dir, app_dir)
+        cmd = f"flower-supernode --insecure --grpc-adapter --superlink {addr} {rel_custom_dir}"

         # use app_dir as the cwd for flower's client app.
         # this is necessary for client_api to be used with the flower client app for metrics logging

